.github/workflows/ai-runner-models.yaml

```diff
@@ -0,0 +1,132 @@
+name: Pull and dockerize huggingface models
+
+on:
+  push:
+    branches:
+      - "main"
+    paths:
+      - "runner/dl_checkpoints.sh"
+      - ".github/workflows/ai-runner-models.yaml"
+  pull_request:
+    paths:
+      - "runner/dl_checkpoints.sh"
+      - ".github/workflows/ai-runner-models.yaml"
+  workflow_dispatch:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.ref_name }}
+  cancel-in-progress: ${{ !((github.ref_type == 'branch' && github.ref_name == 'main') || github.ref_type == 'tag') }}
+
+jobs:
+  download_batch_models:
```

Comment:
batch just [downloads the models](https://github.com/livepeer/ai-runner/blob/main/runner/dl_checkpoints.sh#L159-L163) so it doesn't need to be built per GPU.

Reply:
both jobs could be made into a single strategy matrix, so that any future common change doesnt need to be copied over.